prime-router/docs/api/components/security_schemes.yml

```diff
@@ -0,0 +1,14 @@
+OktaAuth:
```

Comment:
I'm not in favor of using "OktaAuth" since we could someday switch to another oauth2 provider.

Reply:
updated